src/drag-and-drop/components/container.tsx

```diff
@@ -275,7 +276,11 @@ export const Container: React.FC<IProps> = ({ authoredState, interactiveState, s
               top: Math.min(canvasHeight - margin, top)
             };
           }
-          return <DraggableItemWrapper key={item.id} item={item} position={position} draggable={!readOnly} />;
+          // When an item is being dragged, temporarily disable all other draggables until it is dropped. This 
+          // ensures the item registers as dropped in a drop zone even if it is dropped on top of another 
+          // draggable that is already in the drop zone.
+          const tempDisable = isOver && itemDragging.item.id !== item.id;
```

Comment:
Out of curiosity and to make sure I'm understanding things, wouldn't it also work to eliminate the `isOver` test, i.e. to always disable items other than the one being dragged, whether or not we're currently over them?

Reply:
@kswenson `isOver` is coming from [`useDrop`](https://react-dnd.github.io/react-dnd/docs/api/use-drop) which means it's describing whether a draggable is over the drop zone, not whether it's over another draggable. So the draggables not being dragged are always disabled when an item is being dragged over the drop zone. 

I probably should've named the variable something like `isOverDropZone` to make that clear.

I had tried not checking `isOver` at one point, but the results were strange. Trying it out again, though, it does seem to work without it. I must've been doing something else wrong when I tried that before.

I've dropped the use of `isOver` in the latest commit. Does that look good?